Breadcrumbs.sublime-settings

```diff
@@ -1,5 +1,7 @@
 {
-"breadcrumb_length_limit": 100,
-"total_breadcrumbs_length_limit": 200,
-"breadcrumbs_separator": " ■ "
+  "breadcrumb_regex": "^([{}\\(\\)\\[\\]\\s])*\\s*(?P<name>[^{\\(\\[]*[^:{\\(\\[\\s])",
```

Comment:
You've mentioned problems with syntax highlighting, but it is not obvious to me if you meant:
a) that Sublime has problems highlighting this line in this file?
b) that Sublime has problems highlighting a line in breadcrumbs.py which contains `default_breadcrumb_regex = settings.get('breadcrumb_regex', u'^([{}\\(\\)\\[\\]\\s])*\\s*(?P<name>[^{\\(\\[]*[^:{\\(\\[\\s])')`
c) that Sublime has problems highlighting everything once the plugin is initialized and starts interfering in some magical way with syntax highlighter?

(Do you know that in Python you can declare a string as "raw" using `r` instead of `u`, which lets you replace double slashes with single slashes?)